app/controllers/questions_controller.rb

```diff
@@ -78,10 +79,11 @@ def audio_prompt
 
     decorated_question = Odk::QuestionDecorator.decorate(@question)
 
-    send_file decorated_question.audio_prompt.path,
-      type: decorated_question.audio_prompt_content_type,
+    send_attachment(
+      decorated_question.audio_prompt,
       disposition: "attachment",
```

Comment:
Maybe make disposition attachment the default to simplify things further? Inline is only rarely used I think?